tests/rust/transparent.rs

```diff
@@ -8,66 +8,271 @@ struct TransparentComplexWrappingStructTuple(DummyStruct);
 #[repr(transparent)]
 struct TransparentPrimitiveWrappingStructTuple(u32);
 
-// Transparent structure wrapping a struct.
+// Transparent struct wrapping a struct.
 #[repr(transparent)]
-struct TransparentComplexWrappingStructure { only_field: DummyStruct }
+struct TransparentComplexWrappingStruct { only_field: DummyStruct }
```

Comment:
I like this patch in general, but these changes which are not related to the changed behavior makes a bit harder to read the patch.